projects/centre_console/src/brake_state_monitor.c

```diff
@@ -0,0 +1,10 @@
+#include "brake_state_monitor.h"
```

Comment:
No tests, no functionality?

Reply:
My bad, I added `pedal_monitor`, it does the same thing...